### ISSUE TITLE
A11y_AzureToolsStorageExplorer_ToggleExplorerPane_ScreenReader: Voiceover announces "menu item" for all the elements in the left pane.

### ISSUE THREAD

**Issue by Suraj1200499** [microsoft/AzureStorageExplorer#2300]
GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #VoiceOver; #Mac;  

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.11.0
MAC Version: Catalina 10.15 

**Repro Steps:**
1. Launch Storage Explorer
2. Start Voice Over.
3. Navigate to Controls(Toggle Explorer, Manage Accounts, Open Connect dialog, Settings, Send feedback) present on left pane using Tab key.
4. Observe Voice Over announcement.

**​Actual:​**
Voiceover announces 'menu item' for all the elements in the left pane.
For eg: Voiceover says, "Toggle Explorer, menu item"

**Same issue is observed throughout the application for this type of controls.**

**Expected:**
Left pane elements should be buttons as user can navigate in the pane using TAB.
For eg: Voice Over should announce, "Toggle Explorer, button".

**UserImpact:**
Screen reader users may confuse with the Voiceover announcement for the elements in the left pane as Voice Over says, "menu item" for all the elements, however, user can navigate through the elements using Tab.

**Comment by MRayermannMSFT** [comment]
Fix checked into master.

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by Suraj1200499** [comment]
@MRayermannMSFT Verified this issue in Microsoft Azure Tools Storage Explorer Application Version: 1.12.0, this issue is no longer reproduced. Hence Closing the bug.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;
